clustered/server/entity/src/test/java/org/ehcache/clustered/server/store/ClusterTierActiveEntityTest.java

```diff
@@ -113,6 +117,9 @@
 public class ClusterTierActiveEntityTest {
 
   private static final KeySegmentMapper DEFAULT_MAPPER = new KeySegmentMapper(16);
+  private static final int MAX_SYNC_CONCURRENCY = 1;
+  private static final ExecutorService SYNC_GETS_EXECUTOR = new ThreadPoolExecutor(0, MAX_SYNC_CONCURRENCY,
```

Comment:
Are entities passive sync-ed in parallel or sequentially? If it's done in parallel we might have introduced a bottleneck. Previously we had one thread per cache... now we have one thread for all the caches. I think given the way the executors are used the equivalent would be `Executors.newCachedThreadPool()`?

Reply:
Caches are synced sequentially.  A single thread should be sufficient no?